yaptide/examples/call_api_example.py

```diff
@@ -182,20 +183,26 @@ def run_simulation_with_files(session: requests.Session, example_dir, json_to_se
                 print(e)
 
 
-def run_simulation_with_rimrock(port: int = 5000):
-    """Example function running simulation on rimrock"""
-    example_dir = os.path.dirname(os.path.realpath(__file__))
-    grid_proxy_path = Path(example_dir, 'grid_proxy')
+def get_grid_proxy_file(dir_path: str) -> str:
+    """Function reading grid_proxy file"""
+    grid_proxy_path = Path(dir_path, 'grid_proxy')
     try:
         with open(grid_proxy_path) as grid_proxy_file:
             grid_proxy = grid_proxy_file.read()
+            return grid_proxy
     except FileNotFoundError:
         print("Generate grid_proxy file by adjusting following command:\n")
         cmd = "read -s p && echo $p | ssh -l <plgusername> ares.cyfronet.pl "
         cmd += r'"grid-proxy-init -q -pwstdin && cat /tmp/x509up_u\`id -u\`"'
         cmd += f" > {grid_proxy_path} && unset p\n"
         print(cmd)
-        return
+        exit(1)
+
+
+def run_simulation_with_rimrock(port: int = 5000):
+    """Example function running simulation on rimrock"""
+    example_dir = os.path.dirname(os.path.realpath(__file__))
+    grid_proxy = get_grid_proxy_file(dir_path=example_dir)
```

Comment:
No need to specify the function parameter name as it has only one named parameter.
```suggestion
    grid_proxy = get_grid_proxy_file(example_dir)
```